src/mlpack/bindings/R/mlpack/R/package.R

```diff
@@ -5,11 +5,9 @@
 #' learning algorithms. mlpack provides these algorithms as simple command-line
 #' programs, C++ classes and bindings for : Python, Julia, Go and R which can
 #' then be integrated into larger-scale machine learning solutions.
-#'
-#' @docType package
+#' 
 #' @name mlpack
 #' @aliases mlpack-package
-#' @author mlpack Team
 #' @importFrom Rcpp evalCpp
 #' @useDynLib mlpack
-NULL
+"_PACKAGE"
```

Comment:
:+1: 

I needed that in some places and truly hate how some package force you to make such minutae changes.